info-compiler/src/test/java/com/google/sps/WebCrawlerTest.java

```diff
@@ -78,6 +82,42 @@ public static void initialize() throws InterruptedException, IOException {
     newsContentExtractor = mock(NewsContentExtractor.class);
     relevancyChecker = mock(RelevancyChecker.class);
     webCrawler = new WebCrawler(datastore, newsContentExtractor, relevancyChecker);
+
+    // {@code customSearchJson} has the following JSON structure (ellipsis represents other
+    // properties not shown). Insert {@code VALID_URL}.
```

Comment:
Is there documentation on the customSearchJson structure?  Maybe add a link to that documentation instead of writing out the structure.

Reply:
Will do!

Reply:
Will do!

Reply:
The closest one I found is this (in the middle of the page): https://developers.google.com/custom-search/v1/reference/rest/v1/Search
However, it doesn't show the specific structure within `items`. Should I still remove the structure or keep it?

Reply:
Nah this is fine as is then.

Reply:
Ok!